Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -276,6 +277,182 @@ void HoI4::Country::convertLeaders(const graphicsMapper& theGraphics, const Vic2
 }
 
 
+void HoI4::Country::convertMonarchIdea(const graphicsMapper& theGraphicsmapper,
+	 Names& names,
+	 Localisation& hoi4Localisations,
+	 const Vic2::Country& sourceCountry)
+{
+	if (!hasMonarchIdea())
+	{
+		return;
+	}
+
+	std::optional<std::string> firstName;
+	std::optional<std::string> surname = sourceCountry.getLastDynasty();
+
+	bool female = false;
+	if (female)
+	{
```

Comment:
Will this ever execute if in line 293 you set female to false?

Reply:
It won't. Where is female imported? How do you determine it?
Do you want me to export female = true in botanical? Wouldn't make sense given the timeframe though.

Reply:
I've opened a new issue (#897) to add this in the future.